jupyter-docker/Dockerfile

```diff
@@ -223,7 +229,9 @@ RUN R -e 'install.packages(c( \
  && R -e 'devtools::install_github("apache/spark@v2.0.2", subdir="R/pkg")' \
  && mkdir -p /home/jupyter-user/.rpackages \
  && echo "R_LIBS=/home/jupyter-user/.rpackages" > /home/jupyter-user/.Renviron \
- && chown -R $USER:users /home/jupyter-user/.rpackages
+ && chown -R $USER:users /home/jupyter-user/.rpackages \
+ && apt-get clean \
```

Comment:
I think this one should be in the previous layer? The previous one does the apt-gets, this one runs the `R install`s.